repos/system_upgrade/common/actors/selinux/selinuxapplycustom/actor.py

```diff
@@ -2,10 +2,10 @@
 import shutil
 
 from leapp.actors import Actor
-from leapp.models import SELinuxModules, SELinuxCustom, SELinuxRequestRPMs
+from leapp.models import SELinuxModules, SELinuxCustom
 from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
 from leapp.libraries.stdlib import run, CalledProcessError
-
+from leapp.libraries.actor import selinuxapplycustom
```

Comment:
Hmm wondering why didn't the linter catch this. Could you please swap L7 and L8? And I'll check how is our linter doing in the meantime)

Reply:
lint_fix make target should help with that. This relays on local master for comparison, so please make sure that not only your pr is rebased against master, but master is also up to date.

Reply:
Or you can apply these changes
```
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/actor.py b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/actor.py
index e0ae41b..7eca815 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/actor.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/actor.py
@@ -3,11 +3,11 @@ import shutil
 
 from leapp import reporting
 from leapp.actors import Actor
-from leapp.models import SELinuxModules, SELinuxCustom
-from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
 from leapp.libraries.actor import selinuxapplycustom
 from leapp.libraries.actor.selinuxapplycustom import BACKUP_DIRECTORY
-from leapp.libraries.stdlib import run, CalledProcessError
+from leapp.libraries.stdlib import CalledProcessError, run
+from leapp.models import SELinuxCustom, SELinuxModules
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
 
 WORKING_DIRECTORY = '/tmp/selinux/'
 
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/libraries/selinuxapplycustom.py b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/libraries/selinuxapplycustom.py
index 95c06bf..887f381 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/libraries/selinuxapplycustom.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/libraries/selinuxapplycustom.py
@@ -1,9 +1,8 @@
-import re
 import os
+import re
 import shutil
 
-from leapp.libraries.stdlib import api, run, CalledProcessError
-
+from leapp.libraries.stdlib import api, CalledProcessError, run
 
 BACKUP_DIRECTORY = "/tmp/leapp-selinux/"
 
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/tests/component_test_selinuxapplycustom.py b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/tests/component_test_selinuxapplycustom.py
index b0ed347..0171f58 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/tests/component_test_selinuxapplycustom.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/tests/component_test_selinuxapplycustom.py
@@ -2,8 +2,8 @@ import os
 
 import pytest
 
-from leapp.models import SELinuxModule, SELinuxModules, SELinuxCustom, SELinuxFacts, SELinuxRequestRPMs
-from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import SELinuxCustom, SELinuxFacts, SELinuxModule, SELinuxModules, SELinuxRequestRPMs
 from leapp.reporting import Report
 
 TEST_MODULES = [
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/actor.py b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/actor.py
index b764a51..23bc9c2 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/actor.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/actor.py
@@ -1,7 +1,7 @@
 from leapp.actors import Actor
-from leapp.models import SELinuxModules, SELinuxCustom, SELinuxFacts, SELinuxRequestRPMs, RpmTransactionTasks
-from leapp.tags import FactsPhaseTag, IPUWorkflowTag
 from leapp.libraries.actor import selinuxcontentscanner
+from leapp.models import RpmTransactionTasks, SELinuxCustom, SELinuxFacts, SELinuxModules, SELinuxRequestRPMs
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
 
 
 class SELinuxContentScanner(Actor):
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/libraries/selinuxcontentscanner.py b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/libraries/selinuxcontentscanner.py
index e2f3de8..d512803 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/libraries/selinuxcontentscanner.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/libraries/selinuxcontentscanner.py
@@ -2,8 +2,8 @@ import os
 import re
 from shutil import rmtree
 
-from leapp.libraries.stdlib import api, run, CalledProcessError
 from leapp.libraries.common.config import version
+from leapp.libraries.stdlib import api, CalledProcessError, run
 from leapp.models import SELinuxModule
 
 # types and attributes that where removed between RHEL 7 and 8
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/tests/component_test_selinuxcontentscanner.py b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/tests/component_test_selinuxcontentscanner.py
index 2a98aa4..956277b 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/tests/component_test_selinuxcontentscanner.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/tests/component_test_selinuxcontentscanner.py
@@ -2,11 +2,11 @@ import os
 
 import pytest
 
-from leapp.snactor.fixture import current_actor_context
-from leapp.models import SELinuxModule, SELinuxModules, SELinuxCustom, SELinuxFacts, SELinuxRequestRPMs
 from leapp.libraries.common.config import mock_configs
-from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import SELinuxCustom, SELinuxFacts, SELinuxModule, SELinuxModules, SELinuxRequestRPMs
 from leapp.reporting import Report
+from leapp.snactor.fixture import current_actor_context
 
 # compat module ensures compatibility with newer systems and is not part of testing
 TEST_MODULES = [
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxprepare/libraries/selinuxprepare.py b/repos/system_upgrade/common/actors/selinux/selinuxprepare/libraries/selinuxprepare.py
index 0ab2623..96e49b4 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxprepare/libraries/selinuxprepare.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxprepare/libraries/selinuxprepare.py
@@ -1,4 +1,4 @@
-from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.libraries.stdlib import api, CalledProcessError, run
 from leapp.models import SELinuxModules
 
 
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/component_test_selinuxprepare.py b/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/component_test_selinuxprepare.py
index 078ee76..105dcb2 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/component_test_selinuxprepare.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/component_test_selinuxprepare.py
@@ -2,12 +2,11 @@ import os
 
 import pytest
 
-from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.libraries.stdlib import api, CalledProcessError, run
 from leapp.models import SELinuxModule, SELinuxModules
 from leapp.reporting import Report
 from leapp.snactor.fixture import current_actor_context
 
-
 TEST_MODULES = [
     ["400", "mock1"],
     ["99", "mock1"],
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/unit_test_selinuxprepare.py b/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/unit_test_selinuxprepare.py
index beeaf3e..0a6f4c9 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/unit_test_selinuxprepare.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/unit_test_selinuxprepare.py
@@ -1,5 +1,5 @@
 from leapp.libraries.actor import selinuxprepare
-from leapp.libraries.stdlib import CalledProcessError, api, run
+from leapp.libraries.stdlib import api, CalledProcessError, run
 from leapp.models import SELinuxModule, SELinuxModules
 
 
diff --git a/repos/system_upgrade/common/models/selinux.py b/repos/system_upgrade/common/models/selinux.py
index 27c01ea..3305691 100644
--- a/repos/system_upgrade/common/models/selinux.py
+++ b/repos/system_upgrade/common/models/selinux.py
@@ -1,4 +1,4 @@
-from leapp.models import Model, fields
+from leapp.models import fields, Model
 from leapp.topics import SystemInfoTopic, TransactionTopic
```